frontend/__tests__/components/RestaurantCard.test.tsx

```diff
@@ -18,12 +18,13 @@ describe('RestaurantCard', () => {
     render(<RestaurantCard restaurant={FAKE_RESTAURANT} />)
 
     expect(screen.getByText(FAKE_RESTAURANT.name)).toBeInTheDocument()
+    expect(screen.getByText(FAKE_RESTAURANT.website_url)).toBeInTheDocument()
     expect(screen.getByText(FAKE_RESTAURANT.address)).toBeInTheDocument()
     expect(screen.getByText(FAKE_RESTAURANT.cuisine)).toBeInTheDocument()
     expect(screen.getByText(FAKE_RESTAURANT.rating)).toBeInTheDocument()
     expect(screen.getByText(FAKE_RESTAURANT.phone)).toBeInTheDocument()
-    expect(screen.getByText(FAKE_RESTAURANT.image_url)).toBeInTheDocument()
-    expect(screen.getByText(FAKE_RESTAURANT.website_url)).toBeInTheDocument()
+    const image = document.querySelector("img") as HTMLImageElement
```

Comment:
i think we should be able to still use screen/react testing library for this instead of document.querySelector --> `screen.getByRole("img")`.

[Example from the docs](https://testing-library.com/docs/queries/byrole/#api)
`
For example <img aria-label="fancy image" src="fancy.jpg" /> will be returned for getByRole('img', { name: 'fancy image' }). 
`

and then we can assert on the src.  In addition, we can also assert on the `alt` text using screen.*ByAltText`